packages/govern-lib/src/utils/NetworkContext.ts

```diff
@@ -0,0 +1,96 @@
+import { Address, Networkish, Network } from '../types'
+import { networkContexts } from '../config'
+import { ErrorInvalidNetwork } from '../errors'
+
+class NetworkContext implements Network {
```

Comment:
@0xGabi I refactored a little bit the `Network` class to separate the idea of a “network” (as in Ethereum network) and a “network context” (Ethereum network + contextual data, i.e. the subgraph URL). Let me know what you think!

Reply:
I think the network context make sense, specially if we start having more contextual data from places like IPFS.